src/fujimn_int.cpp

```diff
@@ -53,15 +53,15 @@ namespace Exiv2 {
 
     //! Sharpness, tag 0x1001
     constexpr TagDetails fujiSharpness[] = {
-        {   0, N_("-4")             },
-        {   1, N_("Soft mode 1/-3") },
-        {   2, N_("Soft mode 2/-2") },
-        {   3, N_("Normal/0")       },
-        {   4, N_("Hard mode 1/+2") },
-        {   5, N_("Hard mode 2/+3") },
-        {   6, N_("+4")             },
-        { 130, N_("-1")             },
-        { 132, N_("+1")             }
+        {   0, N_("-4")               },
+        {   1, N_("-3 (Soft mode 1)") },
+        {   2, N_("-2 (Soft mode 2)") },
```

Comment:
This is a bit controversial (not symmetrical): soft 1 = -3 but hard 1 = +2; I would expect soft/hard 1 = -/+2 and soft/hard 2 = -/+3... any way to confirm these?